lib/database_transform/tasks/transform.rake

```diff
@@ -16,13 +16,17 @@ class DatabaseTransform::Transform
   private
 
   def require_schema
-    schema_file = @schema.underscore
-    begin
-      return require(File.join(Rails.root, 'db', 'transforms', schema_file))
-    rescue LoadError
+    schema_name = @schema.underscore
+    schema_files = [
+        File.join(Rails.root, 'db', 'transforms', schema_name),
```

Comment:
Which should have higher priority, the one in the folder or the one outside the folder?